libs/ui/tabs/helm/src/lib/hlm-tabs-trigger.directive.ts

```diff
@@ -1,27 +1,20 @@
-import { Directive, Input, computed, inject, input } from '@angular/core';
+import { Directive, computed, input } from '@angular/core';
 import { hlm } from '@spartan-ng/ui-core';
 import { BrnTabsTriggerDirective } from '@spartan-ng/ui-tabs-brain';
 import { ClassValue } from 'clsx';
 
 @Directive({
 	selector: '[hlmTabsTrigger]',
 	standalone: true,
+	hostDirectives: [{ directive: BrnTabsTriggerDirective, inputs: ['brnTabsTrigger: hlmTabsTrigger', 'disabled'] }],
 	host: {
 		'[class]': '_computedClass()',
 	},
-	hostDirectives: [BrnTabsTriggerDirective],
 })
 export class HlmTabsTriggerDirective {
-	private readonly _brn = inject(BrnTabsTriggerDirective);
+	public readonly triggerFor = input.required<string>({ alias: 'hlmTabsTrigger' });
 
-	@Input('hlmTabsTrigger')
-	set triggerFor(key: string) {
-		if (this._brn) {
-			this._brn.triggerFor = key;
-		}
-	}
-
-	private readonly _userClass = input<ClassValue>('', { alias: 'class' });
+	public readonly _userClass = input<ClassValue>('', { alias: 'class' });
```

Comment:
Minor/nit: What do you think about using `protected` here? (Only IF applicable?)

```suggestion
	protected readonly _userClass = input<ClassValue>('', { alias: 'class' });
```

> change _userClass to public, vscode shows errors for class in template otherwise

`protected` effectively behaves like private (to ViewChild-adjacent accessors) while allowing access to the variable in the template. BUT this may be complete irrelevant to your context, I can't really see from the diff and don't have time to delve deeper.

Reply:
Tried `protected` for `_userClass` input and the following template issue is shown. Seems like inputs have to be `public` for correct intellisense.

![CleanShot 2024-02-22 at 10 46 57](https://github.com/goetzrobin/spartan/assets/8985933/ee5983b4-d838-4f40-8ead-5078f7efa507)